datalad/core/distributed/tests/test_clone.py

```diff
@@ -1172,3 +1172,38 @@ def test_gin_cloning(path):
     eq_(result[0]['path'], op.join(ds.path, annex_path))
     ok_file_has_content(op.join(ds.path, annex_path), 'two\n')
     ok_file_has_content(op.join(ds.path, git_path), 'one\n')
+
+
+@with_tree(tree={"special": {"f0": "0"}})
+@serve_path_via_http
+@with_tempfile(mkdir=True)
+def test_fetch_git_special_remote(url_path, url, path):
+    url_path = Path(url_path)
+    path = Path(path)
+    ds_special = Dataset(url_path / "special").create(force=True)
+    if ds_special.repo.is_managed_branch():
+        # NEEDSWORK: git-annex-init fails in the first clone call below when
```

Comment:
Not that we are actively looking for them, but I would have annotated it with a more conventional TODO instead of NEDSWORK so that there is at least a better chance it would get some attention.

Reply:
Updated.

Reply:
thank you! am I correct that this is something we should report to @joeyh ?

Reply:
I don't know.  I would need to look into it more.